### ISSUE TITLE
Static quantization+per_channel is wrong for MobileNetV3 

### ISSUE THREAD

**Issue by shinh** [microsoft/onnxruntime#11415]
**Describe the bug**

When we evaluate MobileNetV3 with ImageNet, the accuracy is fairly low (~54%) when both static quantization and `per_channel=True` are used. With `per_channel=False`, accuracy was ~69%.

**Urgency**
none

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 20.04
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.11.0
- Python version: 3.8.10
- Visual Studio version (if applicable): n/a
- GCC/Compiler version (if compiling from source): n/a
- CUDA/cuDNN version: 11.5
- GPU model and memory:GeForce GTX 1060 3GB

**To Reproduce**

As sharing the whole evaluation script is not easy, let me share a smaller script which reproduces the problem:

```
import numpy as np

import onnx
import onnxruntime as ort
from onnxruntime import quantization

import torch
import torchvision


class MobileNetV3Conv(torch.nn.Module):
    def __init__(self):
        super().__init__()
        m = torchvision.models.mobilenet_v3_large(pretrained=True)
        self.conv_bn_relu = m.features[6].block[1]

    def forward(self, x):
        return self.conv_bn_relu(x)


class SingleSampleDataLoader(quantization.CalibrationDataReader):
    def __init__(self, sample):
        self.sample = sample

    def get_next(self) -> dict:
        result = self.sample
        self.sample = None
        return result


model_name = "test.onnx"
m = MobileNetV3Conv().eval()
x = torch.rand(1, 120, 28, 28)
torch.onnx.export(m, x, model_name, opset_version=14,
                  input_names=["input"],
                  output_names=["output"])

y_float = m(x)

x = x.detach().cpu().numpy()
y_float = y_float.detach().cpu().numpy()

providers = ["CPUExecutionProvider"]
sess = ort.InferenceSession(model_name, providers=providers)
input = {"input": x}
y, = sess.run(["output"], input)
err = np.linalg.norm(y - y_float)
print(f"float (ort) err={err}")


for static in (False, True):
    for per_channel in (False, True):
        quant_model_name = f"test_quant_{static}_{per_channel}.onnx"
        if static:
            data_loader = SingleSampleDataLoader(input)
            quantization.quantize_static(model_name, quant_model_name,
                                         data_loader,
                                         per_channel=per_channel,
                                         activation_type=quantization.QuantType.QUInt8,
                                         weight_type=quantization.QuantType.QUInt8)
        else:
            quantization.quantize_dynamic(model_name, quant_model_name,
                                          per_channel=per_channel,
                                          weight_type=quantization.QuantType.QUInt8)

        sess = ort.InferenceSession(quant_model_name, providers=providers)
        y, = sess.run(["output"], input)
        err = np.linalg.norm(y - y_float)
        print(f"quant static={static} per_channel={per_channel} err={err}")

        model = onnx.load(quant_model_name)
        for init in model.graph.initializer:
            if init.name.endswith("_quantized") and list(init.dims) == [120]:
                print("quantized bias", onnx.numpy_helper.to_array(init).min())
```

**Expected behavior**

On my environment (onnxruntime 1.11.0, onnx 1.10.1, torch 1.11, torchvision 0.12.0, numpy 1.22.3), the program above output:

```
float (ort) err=7.222540443763137e-05
quant static=False per_channel=False err=12.014911651611328
quant static=False per_channel=True err=12.014911651611328
quant static=True per_channel=False err=12.713019371032715
quantized bias -9817
quant static=True per_channel=True err=39.83659744262695
quantized bias -2147483648
```

You see the largest error with `static=True` and `per_channel=True`. I would expect a smaller error. Also, `quantized bias` at the last line has INT_MIN, which would clearly indicate there is an overflow/underflow.

**Additional context**

I think the problem happens here: https://github.com/microsoft/onnxruntime/blob/49d7050b88338dd57839159aa4ce8fb0c199b064/onnxruntime/python/tools/quantization/onnx_quantizer.py#L759

The convolution in pretrained MobileNetV3 has very small weights. For example

```
# Note `test.onnx` is created by the script above
>>> w=onnx.numpy_helper.to_array(onnx.load("test.onnx").graph.initializer[0])
>>> w[23]
array([[[-1.4411272e-18,  3.1880583e-19,  3.1301679e-18,  2.3695221e-18,
          1.4765194e-18],
        [ 6.2421381e-19,  2.6099835e-18,  2.3126957e-18,  3.7413586e-18,
          1.7676403e-18],
        [ 1.1990098e-18,  3.0709601e-18,  3.6974250e-18,  5.3328185e-18,
          3.3157487e-18],
        [-8.0679133e-19,  4.4256888e-18,  1.4706940e-18,  3.7078760e-18,
          3.0225815e-19],
        [ 7.0804898e-19,  1.4276609e-18,  9.8460058e-20,  2.2756359e-20,
          1.2236125e-18]]], dtype=float32)
```

This results in very small `weight_scale` and thus quantized bias becomes large because `bias_scale` is defined by

```
bias_scale = input_scale * weight_scale * beta
```

For some reason, `np.array(1e20).astype(np.int32)` becomes INT_MIN and this is why my script shows `quantized bias -2147483648` at the last line.

I'm not sure what it the right fix for this issue. I think `per_channel` quantization for bias would not make much sense. I made a change like

```
            _, _, bias_zero_point, bias_scale, quantized_data = quantize_data(
                bias_data.flatten().tolist(),
                onnx.TensorProto.INT8,
                True)
            assert bias_zero_point == 0
            bias_scale = np.array(bias_scale)
```

to make `bias_scale` a scalar independent from `weight_scale`. This worked nicely for `CUDAExecutionProvider` (got ~70% accuracy) but didn't work for `CPUExecutionProvider`, so I'm guessing the implementation of `CPUExecutionProvider` depends on the fact `bias_scale` is computed by `input_scale * weight_scale`.
